src/internal/operators/map.ts

```diff
@@ -44,19 +42,17 @@ import { OperatorSubscriber } from './OperatorSubscriber';
  * @name map
  */
 export function map<T, R>(project: (value: T, index: number) => R, thisArg?: any): OperatorFunction<T, R> {
-  return (source: Observable<T>) =>
-    lift(source, function (this: Subscriber<R>, source: Observable<T>) {
-      const subscriber = this;
```

Comment:
This best outlines what changed in each file....

```ts
return (source: Observable<T>) =>	 
    lift(source, function (this: Subscriber<R>, source: Observable<T>) {	
      const subscriber = this;
```

becomes:

```ts
 return operate((source, subscriber) => {
```

The same work is done, it's just more concise.